Fixed error with logging message in SyncNetworkDataIPAddress.update()